src/DotNetOutdated.Core/Services/DependencyGraphService.cs

```diff
@@ -36,6 +34,7 @@ public async Task<DependencyGraphSpec> GenerateDependencyGraphAsync(string proje
                 "/p:TreatWarningsAsErrors=false",
                 "/t:Restore,GenerateRestoreGraphFile",
                 $"/p:RestoreGraphOutputPath=\"{dgOutput}\"",
+                $"/p:RuntimeIdentifiers={runtime}"

```

Comment:
Allow for multiple comma separated rids
```suggestion
                $"/p:RuntimeIdentifiers=\"{runtime}\""

```

Reply:
Thanks for it I missed it